packages/bsp/helios4/armbian-motd

```diff
@@ -1,9 +1,13 @@
 # add space-separated list of MOTD script names (without number) to exclude them from MOTD
 # Example:
 # MOTD_DISABLE="header tips updates"
+# configure display of Interfaces/IP addresses in 30-armbian-sysinfo
+# Example:
+# IP_AND_INTERFACE_NAMES="none one list table"
 
 MOTD_DISABLE=""
 ONE_WIRE="" # yes = show 1-wire temperature sensor if attached 
+IP_AND_INTERFACE_NAMES="none"
```

Comment:
But where is this parameter included?

This is correct file for changes:
https://github.com/armbian/build/blob/master/packages/bsp/common/etc/default/armbian-motd.dpkg-dist

Reply:
Sorry Igor, I thought I was making that change in my private fork. It's no way ready for for review. I plan to use the token(s) in 30-armbian-sysinfo to control the control the sysinfo display.
"none" suppress both IP and and interface display.
"one" Single IP and Interface display
"list" output IP and interface display as a space separated list.
"table" output IP and interface one per line
probably "iponly" suppress the interface reports
But I intend to test it on my systems before it gets anywhere further.
Harry

Reply:
OK, don't worry.